lib/internal/modules/helpers.js

```diff
@@ -131,13 +159,22 @@ function makeRequireFunction(mod, redirects) {
     };
   }
 
+  /**
+   * The `resolve` method that gets attached to module-scope `require`.
+   * @param {string} request
+   * @param {Parameters<Module['_resolveFilename']>[3]} options
+   */
   function resolve(request, options) {
     validateString(request, 'request');
     return Module._resolveFilename(request, mod, false, options);
   }
 
   require.resolve = resolve;
 
+  /**
+   * The `paths` method that gets attached to module-scope `require`.
+   * @param {string} request
```

Comment:
I'm confused. If we assume that `request` is a string, shouldn't we run `validateString` in the function that calls this function?

Reply:
Maybe? But this PR isn’t aiming to edit any content, just add JSDoc.

In this particular case maybe it’s already validated before this function is called, so there’s no need to validate again.